lib/ExpensiMark.js

```diff
@@ -139,8 +139,8 @@ export default class ExpensiMark {
                  * Use [\s\S]* instead of .* to match newline
                  */
                 name: 'italic',
-                regex: /(?!_blank")\b_((?!\s)[\s\S]*?\S)_\b(?![^<]*(<\/pre>|<\/code>|<\/a>|_blank))/g,
-                replacement: (match, g1) => (g1.includes('<pre>') ? match : `<em>${g1}</em>`),
+                regex: /(?!_blank")([^\W_]?)_((?![\s_])[\s\S]*?[^\s_])_(?![^\W_])(?![^<]*(<\/pre>|<\/code>|<\/a>|_blank))/g,
```

Comment:
I don't think we need this capture group here. We aren't doing anything with the captured characters, and it would let us avoid changing the `replacement`. The test suite passes locally, and it captures the same test strings mentioned in the issue without this group
```suggestion
                regex: /(?!_blank")[^\W_]?_((?![\s_])[\s\S]*?[^\s_])_(?![^\W_])(?![^<]*(<\/pre>|<\/code>|<\/a>|_blank))/g,
```